### PR TITLE
Doc default behavior of install tests

### PR DESCRIPTION
**Pull request by markcmiller86**
Adding minor verbiage to document default behavior of install tests.

### CODE REVIEW

File: lib/spack/docs/packaging_guide.rst [hunk 1]
```diff
@@ -4016,8 +4016,11 @@ the package being tested when ``installed`` is ``True``.
 The executable runs in ``work_dir``, when specified, using the provided
 ``options``. The return code is checked against the ``status`` argument,
 which can be an integer or list of integers representing status codes
-corresponding to successful execution. Spack also checks that every string
-in ``expected`` is a regex matching part of the output from the test run.
+corresponding to successful execution (e.g. ``status=[0,3,7]``).
+Spack also checks that every string in ``expected`` is a regex matching
+part of the output from the test run (e.g.
+``expected=['completed succesfully', 'converged in']``). Default behavior
```
Comment (tldahlgren) [inline lib/spack/docs/packaging_guide.rst]:
```suggestion
``expected=['completed successfully', 'converged in']``). Default behavior
```

**Comment by markcmiller86** [comment]
Hey, I've never seen a "commit suggestion" button before. How did you do that?

File: lib/spack/docs/packaging_guide.rst [hunk 2]
```diff
@@ -4016,8 +4016,11 @@ the package being tested when ``installed`` is ``True``.
...
+is to behave as though ``status=[0,]`` is specified.
```
Comment (tldahlgren) [inline lib/spack/docs/packaging_guide.rst]: Maybe a nit, but why include "," in the list?  (I'm used to seeing inclusion of a comma when one is initializing a set, not a list.)

Comment (markcmiller86) [reply lib/spack/docs/packaging_guide.rst]: Maybe my python ignorance. Thought it was needed for lists too. Will fix

**Comment by tldahlgren** [comment]
> Hey, I've never seen a "commit suggestion" button before. How did you do that?

It's the first option on the left (after `Preview`) when you add feedback.  It looks like a document with a "+" and "-" in the body.

**Review by tldahlgren — APPROVED** [review]
LGTM.  Approve under assumption all the checks pass *though* I would prefer the change to remove the comma from list initialization.

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Changing to request changes to reflect preference for comma removal in other comment.

**Comment by markcmiller86** [comment]
> Changing to request changes to reflect preference for comma removal in other comment.

Sorry. Don't follow. What change are you requesting?

**Comment by tldahlgren** [comment]
> > Changing to request changes to reflect preference for comma removal in other comment.
> 
> Sorry. Don't follow. What change are you requesting?

https://github.com/spack/spack/pull/21309/files/32e547bad3a06dfc10930c35e28e82f6428408ed#r565000351

**Comment by markcmiller86** [comment]
> > > Changing to request changes to reflect preference for comma removal in other comment.
> > 
> > 
> > Sorry. Don't follow. What change are you requesting?
> 
> https://github.com/spack/spack/pull/21309/files/32e547bad3a06dfc10930c35e28e82f6428408ed#r565000351

Are you not seeing https://github.com/spack/spack/pull/21309/commits/1b10dd568ff41d5ac8d978e29a525c44c972acaf

**Comment by tldahlgren** [comment]
> > > > Changing to request changes to reflect preference for comma removal in other comment.
> > > 
> > > 
> > > Sorry. Don't follow. What change are you requesting?
> > 
> > 
> > https://github.com/spack/spack/pull/21309/files/32e547bad3a06dfc10930c35e28e82f6428408ed#r565000351
> 
> Are you not seeing [1b10dd5](https://github.com/spack/spack/commit/1b10dd568ff41d5ac8d978e29a525c44c972acaf)

Interesting.  I was not at the time.

**Review by tldahlgren — APPROVED** [review]

**Comment by tldahlgren** [comment]
Thank you for your contribution!

**Pull request merged**